docs/concept-exercises.md

```diff
@@ -93,9 +93,68 @@ See the C# floating-point-numbers exercise's [design.md file][csharp-docs-design
 
 ### `.meta/config.json`
 
-This file contains meta information on the exercise, which currently includes the exercise's contributors, and (optionally) language version requirements.
-
-See the C# floating-point-numbers exercise's [config.json file][csharp-docs-config.json] for an example.
+This file contains meta information on the exercise, which may include
+
+- the exercise's contributors
+- the exercise's authors
+- which exercise it was forked from
+- language version requirements
+
+#### Example
+
+Assume that the user FSharpForever has written an exercise called `basics` for the F# track. PythonProfessor adapts the exercise for the Python track. Later on, the user PythonPerfection improves the exercise.
+
+##### `.../python/.../basics/.meta/config.json`
+
+```json
+{
+    "contributors": [
+        {
+            "github_username": "PythonProfessor",
+            "exercism_username": "PythonProfessor"
+        },
+        {
+            "github_username": "PythonPerfection",
+            "exercism_username": "PythonPerfection"
+        }
+    ],
+    "authors": [
+        {
+            "github_username": "FSharpForever",
+            "exercism_username": "FSharpForever"
+        }
+    ],
+    "forked_from": ["fsharp/basics"],
```

Comment:
Interesting to have this an array. Do we ever expect to fork from multiple exercises? @iHiD @SleeplessByte? Theoreticallt one could, so I don't mind this being an array.

Reply:
I don't expect them to be forked from different exercises but it's possible that one forks for example `fsharp/basics` but also uses additions that were made in the `typescript/basics` version of the exercise. This isn't necessarily the same as a transitive dependency. I'm not sure how common this will be, but making it an array allows it with, afaict, few downsides.

Reply:
Agreed. It is better to be prepared for this possiblity.